apps/system-e2e/src/support/index.d.ts

```diff
@@ -2,6 +2,46 @@
 // load type definitions that come with Cypress module
 /// <reference types="cypress" />
 
+interface CypressFn {
+  (): Cypress.Chainable<JQuery<HTMLElement>>
+}
+
+interface IDSLogin {
+  phoneNumber: string
+  urlPath: string
+}
+
+// NOTE: Copied from the runtime generated scheme at app/air-discount-scheme/web/graphql/schema.tsx
+//       Is there a pattern to import generated types?
+type User = {
+  __typename?: 'User'
+  nationalId: Scalars['ID']
+  name: Scalars['String']
+  mobile?: Maybe<Scalars['String']>
+  role: Scalars['String']
+  fund?: Maybe<Fund>
+  meetsADSRequirements: Scalars['Boolean']
+  flightLegs: Array<FlightLeg>
+}
+
+type ConnectionDiscountCode = {
+  __typename?: 'ConnectionDiscountCode'
+  code: Scalars['ID']
+  flightId: Scalars['String']
+  flightDesc: Scalars['String']
+  validUntil: Scalars['String']
+}
+
+type Discount = {
+  __typename?: 'Discount'
+  nationalId: Scalars['ID']
+  discountCode: Scalars['String']
+  connectionDiscountCodes: Array<ConnectionDiscountCode>
+  expiresIn: Scalars['Float']
+  user: User
+}
+// NOTE: end
```

Comment:
Is it possible to import as a library from the `gen/fetch` client?

Reply:
good question, I will take a look

Reply:
The types were available in `libs/shared/air-discount-scheme`

Thanks for the getting me on the right track 👍🏼